recipes/grad-cam/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "grad-cam" %}
+{% set version = "1.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grad-cam-{{ version }}.tar.gz
+  sha256: 26e8f0f2e0e28e7c4f486f26886ebd43f1ecff58a015cf065f8f697551944c80
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - libgdal
```

Comment:
This only seems to be needed at runtime.

Reply:
Done.